src/twp-editor-plugin/files/plugin-state-templates/reducer.ts

```diff
@@ -0,0 +1,8 @@
+import { <%= classify(name) %>PluginState } from './types';
+import { <%= classify(name) %>ActionTypes, <%= classify(name) %>Action } from './actions';
+
+export function reducer(state: <%= classify(name) %>PluginState, action: <%= classify(name) %>Action): <%= classify(name) %>PluginState {
+  switch(action.type) {
```

Comment:
Should we add the sample action here as well?
```
case <%= classify(name) %>ActionTypes.SOME_ACTION:
  return { ...state, /* fill this in */ };
```

Reply:
Yeah good idea